bin/git-create-branch

```diff
@@ -44,9 +51,16 @@ then
 	rm -f "$stderr"
 	if [ $REMOTE_EXIT -eq 0 ]
 	then
-		git push $REMOTE HEAD:refs/heads/$BRANCH
-		git fetch $REMOTE
-		git checkout --track -b $BRANCH $REMOTE/$BRANCH
+		if [[ -n $START_POINT ]]
+		then
```

Comment:
```suggestion
		if [[ -n $START_POINT ]]; then
```

Reply:
Update.

And a question I want to ask is how to choose the type of using the style of the `if then`:

Style A:
```
if [[ ... ]]; then
   ...
fi
```

Style B:
```
if [[ ... ]]
then
    ...
fi
```

Both of them exist in the file. That confuse me a little.

Thanks :D.